### ISSUE TITLE
Need help in adding the solutions for Codechef LearnDSA

### ISSUE THREAD

**Issue by satyajitghana** [satyajitghana/Codechef-LearnDSA#1]
Add solutions to Codechef LearnDSA

create a new issue for the stuff you would want to work on, so we don't have clashes.

NOTE: refrain from adding solutions from ongoing contests

This should be a pretty good repository for hacktoberfest :)

**Comment by Zeeshan-2k1** [comment]
I want to take this

**Comment by charchitmangal** [comment]
Hello! I want to work this

**Comment by satyajitghana** [comment]
sure guys go ahead @Zeeshan-2k1 @charchithere , add solutions, do some documentation !

**Comment by poojitha2002** [comment]
Hello sir! I would like to work for this

**Comment by satyajitghana** [comment]
hi @poojitha2002 , yeah go ahead, hopefully your work doesn't clash with people already contributing, you can take a look at this repo as well: https://github.com/satyajitghana/project-euler/issues/1

**Comment by dyutimangal** [comment]
I would like to contribute.

**Comment by satyajitghana** [comment]
@Zeeshan-2k1 @charchithere @poojitha2002 @dyutimangal  can you guys mention what you are working on, so we don't have clashes.

best would be to create a new issue for the stuff you are working on.

**Comment by Riya-code** [comment]
I would like to take it.

**Comment by princekalra** [comment]
hey can i contribute on this

**Comment by satyajitghana** [comment]
@princekalra yeah go ahead, first create an issue with what you are working on, and on the pull request reference that issue.

**Comment by princekalra** [comment]
> @princekalra yeah go ahead, first create an issue with what you are working on, and on the pull request reference that issue.

 can i push the soln of any codechef contest question

**Comment by satyajitghana** [comment]
> > @princekalra yeah go ahead, first create an issue with what you are working on, and on the pull request reference that issue.
> 
>  can i push the soln of any codechef contest question

Yeah, as long as it comes under Codechef LearnDSA and it's not an ongoing contest
